examples/tutorials/e2e-development/chat-with-pdf.md

```diff
@@ -310,7 +326,7 @@ the other choices, please refer to [flow deploy docs](https://microsoft.github.i
 Use the command below to build a flow as docker format app:
 
 ```bash
-pf flow build --source . --output dist --format docker
+pf flow build --source . --output build --format docker
```

Comment:
I think this should be a resolve conflict issue? 
we intentionally changed the output dir from build to dist in another PR.